syft/cpe/merge_cpes.go

```diff
@@ -23,3 +24,24 @@ func Merge(a, b []CPE) (result []CPE) {
 	sort.Sort(BySpecificity(result))
 	return result
 }
+
+// MergeSourcedCPEs returns unique SourcedCPEs that are found in A or B
+// Two SourcedCPEs are identical if their source and normalized string are identical
+func MergeSourcedCPEs(a, b []CPE) []CPE {
```

Comment:
I think this can be simplified to just `Merge()` since it's already in the `cpe` package (thus `cpe.Merge(...)`)